.infrastructure/pvc.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistantVolumeClaim
```

Comment:
The kind 'PersistantVolumeClaim' is misspelled. It should be 'PersistentVolumeClaim'.